probe-rs/src/architecture/arm/communication_interface.rs

```diff
@@ -320,7 +320,9 @@ impl UninitializedArmProbe for ArmCommunicationInterface<Uninitialized> {
         mut self: Box<Self>,
         sequence: Arc<dyn ArmDebugSequence>,
     ) -> Result<Box<dyn ArmProbeInterface>, ProbeRsError> {
+        log::debug!("hehexd");
         sequence.debug_port_setup(&mut self.probe)?;
+        log::debug!("kekekekeke");
```

Comment:
:rofl:

Reply:
Gnah I always forget to remove those before committing ... Fixed!